app/controllers/screenings_controller.rb

```diff
@@ -63,25 +65,20 @@ def destroy
 
   private
 
-    def set_screening
-      :set_movie
-      @screening = @movie.screenings.by_user(current_user).find(params[:id])
-    end
-
-    def set_movie
-      if params[:tmdb_id].present?
-        @tmdb_id = params[:tmdb_id]
-        unless Movie.exists?(tmdb_id: @tmdb_id)
-          tmdb_handler_add_movie(@tmdb_id)
-        end
-        @movie = Movie.find_by(tmdb_id: @tmdb_id)
-      else
-        @movie = Movie.friendly.find(params[:movie_id])
-      end
-    end
+  def set_screening
+    :set_movie
```

Comment:
🙈 Da heck? I know this was already here, but we should probably 🪓 it since it's not doing anything.